### PR TITLE
add header option

### PR DESCRIPTION
**Pull request by momom-i**
I tried to work on the new feature mentioned in #79 to set the header option.

### CODE REVIEW

File: fuzzyfinder.go [hunk 1]
```diff
@@ -144,22 +146,38 @@ func (f *finder) _draw() {
...
 	}
-	f.term.ShowCursor(promptLinePad+f.state.cursorX, height-1)
+	f.term.ShowCursor(promptLinePad+f.state.cursorX, maxHeight-1)
+
+	maxHeight -= 1
+
+	// Header line
+	if len(f.opt.header) > 0 {
+		w = 0
+		for _, r := range []rune(runewidth.Truncate(f.opt.header, maxWidth-3, "...")) {
```
Comment (ktr0731) [inline fuzzyfinder.go:159]: We use two dots to represent omitted string.
See: https://github.com/ktr0731/go-fuzzyfinder/blob/2bff5cea2b3a88d53c3c7401bafb0f1dd8347b70/fuzzyfinder.go#L229-L230

```suggestion
		for _, r := range []rune(runewidth.Truncate(f.opt.header, maxWidth-2, "..")) {
```

File: fuzzyfinder.go [hunk 2]
```diff
@@ -144,22 +146,38 @@ func (f *finder) _draw() {
...
 
 		// Add a space between '>' and runes.
-		f.term.SetContent(promptLinePad+w, height-1, r, nil, style)
+		f.term.SetContent(promptLinePad+w, maxHeight-1, r, nil, style)
 		w += runewidth.RuneWidth(r)
 	}
-	f.term.ShowCursor(promptLinePad+f.state.cursorX, height-1)
+	f.term.ShowCursor(promptLinePad+f.state.cursorX, maxHeight-1)
+
+	maxHeight -= 1
```
Comment (ktr0731) [inline fuzzyfinder.go:154]:
```suggestion
	maxHeight--
```

**Review by ktr0731 — COMMENTED** [review]
We're almost there!

File: fuzzyfinder.go (hunk 1 above)
Comment (momom-i) [reply fuzzyfinder.go:159]: I understand, thank you.

**Review by ktr0731 — APPROVED** [review]
LGTM 👍 
Thanks for your contribution!

**Pull request merged**